plugins/reporters/web-app-template/src/components/SummaryView.js

```diff
@@ -373,87 +373,78 @@ class SummaryView extends React.Component {
                             <Col span={22} offset={1}>
                                 <Tabs tabPosition="top" className="ort-tabs-summary-overview">
                                     {
-                                        webAppOrtResult.hasRuleViolations()
-                                        && (
-                                            <TabPane
-                                                tab={(
-                                                    <span>
-                                                        <ExceptionOutlined />
-                                                        Rule Violations (
-                                                        {
-                                                            ruleViolations.length !== unresolvedRuleViolations
-                                                            && `${unresolvedRuleViolations}/`
-                                                        }
-                                                        {ruleViolations.length}
-                                                        )
-                                                    </span>
-                                                )}
-                                                key="ort-summary-rule-violations-table"
-                                            >
-                                                <RuleViolationsTable
-                                                    onChange={
-                                                        SummaryView.onChangeRuleViolationsTable
+                                        <TabPane
```

Comment:
Commit message nits:
* "Always **show the** ... issue**s** ... tab**s**."
* "**the** one hand"
* "same tab**s**"

Reply:
Thank you. I have updated the message.

Reply:
Hi Martin, I implemented your comments.